tests/dlt_message_unit_tests.py

```diff
@@ -146,7 +146,7 @@ def test_from_bytes_log_multipayload(self):
         assert_equal(msg.ctid, "CPUS")
         assert_equal(msg.ecuid, "MGHS")
         assert_equal(msg.tmsp, 3.8088)
-        assert_equal(msg.payload_decoded, "4 online cores\n")
+        assert_equal(msg.payload_decoded, "4 online cores ")
```

Comment:
Just curious. Why do we need to change this comments?

Reply:
Not quite sure, but there have been some changes between the previous version and this one namely on dli_message_payload